inlang/source-code/sdk-v2/src/types/plugin.ts

```diff
@@ -117,17 +137,9 @@ export type ResolvePlugins2Function = (args: {
  * The API after resolving the plugins.
  */
 export type ResolvedPlugin2Api = {
-	// TODO SDK-V2 check how to setup loadMessages
-	// loadMessages: (args: {
-	// 	settings: ProjectSettings2
-	// 	nodeishFs: NodeishFilesystemSubset
-	// }) => Promise<Message[]> | Message[]
-	// TODO SDK-V2 check how to setup saveMessages
-	// saveMessages: (args: {
-	// 	settings: ProjectSettings2
-	// 	messages: Message[]
-	// 	nodeishFs: NodeishFilesystemSubset
-	// }) => Promise<void> | void
+	toBeImportedFiles: Array<Plugin2["toBeImportedFiles"]>
+	importFiles: Array<Plugin2["importFiles"]>
+	exportFiles: Array<Plugin2["exportFiles"]>
```

Comment:
@martin-lysk @samuelstroschein how should the resolved plugin API look like? do we support multiple exporter / importer?

Reply:
We don't need resolved plugin api anymore as we don't merge APIs together.